.gitignore

```diff
@@ -4,3 +4,7 @@ out
 .config
 .config.old
 klippy/.version
+klippy/extras/_ToDos
+klippy/extras/tmc.py
+klippy/extras/gcode_shell_command.py
+.vscode
```

Comment:
I do not think that these are valid changes. E.g. `gcode_shell_command.py` is an unofficial "extra" and not part of Klipper main-line

Reply:
Correct. Forget about these changes, they are not meant for merging. I am testing the minimal version of the zhop code at the moment. Once it is working, this PR will be closed, and a "clean" PR will be submitted. Cheers. Florian